src/itinerary/itinerary.service.ts

```
@@ -781,9 +925,53 @@ export class ItineraryService {
+        !itinerary.access.some((access) => access.userId === user?.id)
+      )
+        throw new ForbiddenException(
+          'You do not have permission to view this itinerary'
+        )
+    }
+
+    const result = {
+      ...itinerary,
+      invitedUsers: itinerary.access.map((access) => access.user),
+    }
+    delete result.access
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)